aiida_registry/fetch_metadata.py

```diff
@@ -46,28 +46,6 @@ def load_plugins_metadata(json_file_path):
         return None
 
 
-def get_last_fetched_version(plugin_name):
```

Comment:
Are you sure this can be removed?

Reply:
I think yes. It was inly used for when pypi verrinn can not be get, and store things to that file not exist. I didn't see different after I remove it. Sorry that I merged it without your review.

Reply:
So, are you saying only packages on PyPI should have a latest release date?

If so, perhaps we should be clear about this in the README, since we are using this to sort plugins

Reply:
It is a yes, the page is not yet finished built after I merge this so it shows the old one with this function. The plug-ins not released don't have this badge.